src/index.js

```diff
@@ -2,5 +2,5 @@
 
 export {default} from './command'
 export {default as Topic} from './topic'
-export {BooleanFlag, StringFlag, NumberFlag, Flag} from './flags'
+export {boolean, string, number, app, remote, Flag} from './flags'
```

Comment:
not sure how the first one would work, do you mean `const {flags} = require('cli-engine-command')`? because that's the same as `const flags = require('cli-engine-command').flags`